tools/alive-tv.cpp

```diff
@@ -0,0 +1,135 @@
+// Copyright (c) 2018-present The Alive2 Authors.
+// Distributed under the MIT license that can be found in the LICENSE file.
+
+#include "llvm/ADT/StringExtras.h"
+#include "llvm/Bitcode/BitcodeReader.h"
+#include "llvm/IR/LLVMContext.h"
+#include "llvm/IR/Function.h"
+#include "llvm/IR/Module.h"
+#include "llvm/IR/LegacyPassManager.h"
+#include "llvm/Support/CommandLine.h"
+#include "llvm/Support/Debug.h"
+#include "llvm/Support/ManagedStatic.h"
+#include "llvm/Support/PrettyStackTrace.h"
+#include "llvm/Support/Signals.h"
+#include "llvm/Support/SourceMgr.h"
+#include "llvm/Support/ToolOutputFile.h"
+
+#include "ir/function.h"
+#include "llvm_util/utils.h"
+#include "smt/smt.h"
+#include "smt/solver.h"
+#include "tools/alive_parser.h"
+#include "util/config.h"
+#include "util/file.h"
+#include <cstdlib>
+#include <iostream>
+#include <optional>
+#include <string_view>
+#include <vector>
+
+using namespace IR;
+using namespace tools;
+using namespace util;
+using namespace std;
+using namespace llvm_util;
+
+static llvm::cl::opt<std::string>
```

Comment:
Would be useful to add these to a new standalone category
```
static llvm::cl::OptionCategory AliveTV("alive-tv options");

static llvm::cl::opt<std::string>
opt_file1(llvm::cl::Positional, llvm::cl::desc("first_bitcode_file"),
    llvm::cl::Required, llvm::cl::value_desc("filename"),
    cl::cat(AliveTV));
```

Reply:
yeah, I'll do this

Reply:
done